src/main/java/atoml/testgen/TestcaseGenerator.java

```diff
@@ -64,7 +67,7 @@ public TestcaseGenerator(Algorithm algorithmUnderTest, List<DataDescription> mor
 		default:
 			// TODO
 			templateEngine = null;
-			System.err.format("unknown framework: %s", algorithmUnderTest.getFramework());
+			System.err.format("unknown framework: %s ", algorithmUnderTest.getFramework());
```

Comment:
undo?

Reply:
this was actually just a formatting fix, there was a space missing in the output when the error was thrown, should i revert?

Reply:
I am confused. The space you added is *after* the %s, shouldn't that just add a trailing space?

My guess is that the problem is different: format does not add a new line (like println), so maybe there should be a %n?

Reply:
Yes, exactly. It outputs the framework then a trailing space and follows it up with Exception in thread "main". A new line looks much better there!